nuxeo-features/nuxeo-admin-center/nuxeo-admin-center-oauth/src/main/java/org/nuxeo/ecm/admin/oauth2/OAuth2ClientsActions.java

```diff
@@ -84,7 +85,14 @@ protected void handleValidationError(FacesContext context, String label) {
 
     public void validateClientId(FacesContext context, UIComponent component, Object value) {
         OAuth2ClientService clientService = Framework.getService(OAuth2ClientService.class);
```

Comment:
I would have moved this just before `if (clientService.hasClient((String) value))`